apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx

```diff
@@ -17,36 +20,58 @@ import { Text } from '_app/shared/text';
 import Alert from '_components/alert';
 import Icon, { SuiIcons } from '_components/icon';
 import LoadingIndicator from '_components/loading/LoadingIndicator';
-import { useAppSelector } from '_hooks';
+import { useAppSelector, useGetObject } from '_hooks';
 import { FEATURES } from '_src/shared/experimentation/features';
 
 export function ValidatorsCard() {
     const accountAddress = useAppSelector(({ account }) => account.address);
     const {
-        data: stakeValidators,
+        data: delegations,
         isLoading,
         isError,
         error,
     } = useGetDelegatedStake(accountAddress || '');
 
+    const { data: validators } = useGetObject(STATE_OBJECT);
+
+    const validatorsData = validators && validatorsFields(validators);
```

Comment:
Why not just have `validatorsFields` take a nullable value so that you don't need to `&&` here?